_pages/introduction.md

```diff
@@ -0,0 +1,124 @@
+
+We wrote this as an informal guide; if this information conflicts with explanations on linked websites such as Regulations.gov and Reginfo.gov, please consider them more authoritative.
+
+**Table of contents**
+
+* <a href="#what-are-regulations">What are regulations and where do they live?</a>
+* <a href="#how-regulations-change">How regulations change (through “rules”)</a>
+* <a href="#structure">The structure of a regulation</a>
+* <a href="#definitions">Definitions of terms used in eRegulations</a>
+
+## <a name="what-are-regulations"></a>What are regulations and where do they live?
+
+### Where regulations come from
+
+When Congress creates a law, it’s a big deal — a tremendous amount of effort goes into each new act of Congress. As a result, Congress usually describes the big picture of the law and gives authority to the relevant federal agencies to deal with the specifics. The agencies describe these details in regulations, which have the force and effect of law. ([The first two answers in this Reginfo.gov FAQ provide details about this process.](http://www.reginfo.gov/public/jsp/Utilities/faq.jsp))
+
+How this gets written down: Congress writes “Acts” or “Statutes” which are cataloged in the [U.S. Code](https://www.gpo.gov/fdsys/browse/collectionUScode.action?collectionCode=USCODE) (USC). Each act or statute can give authority to specific federal regulatory agencies to write regulations. When the agencies write regulations, they are cataloged separately in the [Code of Federal Regulations](https://www.gpo.gov/fdsys/browse/collectionCfr.action?collectionCode=CFR) (CFR).
+
+### Who is responsible for regulations
+
+Federal agencies are responsible for maintaining and enforcing their own regulations. However, the Government Printing Office is the central repository for agency regulations. Generally, these two entities agree on what the “regulation” says, but if they ever differ (for example due to errors), the GPO “wins”.
+
+### Where regulations are published
+
+The Government Printing Office publishes an official printed paper Code of Federal Regulations and [official digital Code of Federal Regulations (on FDsys)](https://www.gpo.gov/fdsys/browse/collectionCfr.action?collectionCode=CFR) once a year.
+
+Throughout the year, GPO publishes changes to regulations through [e-CFR (Electronic Code of Federal Regulations)](http://www.ecfr.gov/), though this is not legally binding.
+
+Proposed changes and final changes effective some time in the future are published in the [Federal Register](https://www.federalregister.gov/) (the “Daily Journal of the United States Government”).
```

Comment:
I think it'd be more accurate to say that the changes to the regulations are published in the Federal Register. eCFR is a manual compilation of those changes, as is effective _today_. Importantly, eCFR doesn't give any sort of indication of what's changed recently.